src/main/java/uk/gov/hmcts/ccd/domain/service/getcase/GetCaseOperation.java

```diff
@@ -6,24 +6,25 @@
 
 public interface GetCaseOperation {
 
-    /**
-     * Execute.
+
+    /* Execute.
      *
      * @param jurisdictionId Case's jurisdiction
      * @param caseTypeId Case's case type
      * @param caseReference 16-digit universally unique case reference
      * @return Optional containing CaseDetails when found; empty optional otherwise
      *
-     * @deprecated Use {@link GetCaseOperation#execute(String)} instead
-     */
-    @Deprecated
+     * @deprecated replaced by {@link GetCaseOperation#execute(String)}
+    */
+    @Deprecated(since = "3.5",forRemoval = false)
```

Comment:
```suggestion
    @Deprecated
```
Do we need these new bits? We have other usages of `@Deprecated` e.g. in `DefaultCaseDefinitionRepository` where Sonar isn't complaining that these are missing.

Reply:
Updated